pkg/controller/controller.go

```diff
@@ -210,12 +217,14 @@ func (c *Controller) Run(stopCh <-chan struct{}) error {
 			return nil
 		case message := <-c.configRefreshChan:
 			// Reload the configuration.
-			conf, confErr := c.provider.BuildConfig()
-			if confErr != nil {
-				c.logger.Errorf("Unable to build dynamic configuration: %v", confErr)
+			topo, err := c.topologyBuilder.Build(c.ignored)
```

Comment:
`s/ignored/ignoredSomething`

Can we rename this variable to explicitly state what we ignore, is it kubernetes resources?